person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
```

Comment:
- [ ] Kindly make sure to add `@id = Random.rand(1..1000)` with age, name and parent permission as its the requirement.